spec/models/authenticator/amazon_spec.rb

```diff
@@ -124,12 +124,12 @@ class FakeGroup < Struct.new(:name)
   end
 
   before(:each) do
-    allow_any_instance_of(described_class).to receive(:aws_connect) { |*args| FakeAmazon.new(user_data, *args) }
+    allow_any_instance_of(described_class).to receive(:aws_connect) { |instance, *args| FakeAmazon.new(user_data, *args) }
```

Comment:
hmm.  This one is strange.  What's the deal with instance here?  How did this work before?  It doesn't look like we're using it, should it be `_instance`?